packages/web-app-files/src/components/SideBar/ActivitiesPanel.vue

```diff
@@ -55,9 +55,9 @@ export default defineComponent({
     const activities = ref<Activity[]>([])
     const activitiesLimit = 200
 
-    const activitiesLimitText = computed(() => {
-      return $gettext('(Showing latest %{activitiesLimit} activities)', {
-        activitiesLimit: activitiesLimit.toString()
+    const activitiesFooterText = computed(() => {
+      return $gettext('Showing %{activitiesCount} activities', {
```

Comment:
Please use `$ngettext` and two strings, one for singular and one for plural.

Reply:
on it my fren!!!!!